### ISSUE TITLE
disableConsoleOutput() is throwing NPE

### ISSUE THREAD

**Issue by eyny** [Hakky54/log-captor#48]
**Describe the bug**
disableConsoleOutput() is throwing NPE. The reason lies in the source code of LogCaptor.java. It is searching the console appender with "console" string which returns null. However, when in debugging mode I called the same method with "CONSOLE" string parameter and voila, it returned the appender successfully. This parameter is gathered from CONSOLE_APPENDER_NAME constant which is always "console".

In summary, getAppender method in LogCaptor.java should search case-insensitively or using both "console" and "CONSOLE" parameters, not just "console".

**Expected behavior**
It should not throw NPE and disable console output successfully.

**Environmental Data:**
- Log-captor: 2.9.0
- Spring Boot: 3.1.0
- Log4j: 2.20.2
- JUnit: 5.9.3
- Mockito: 5.2.0
- Java Version: 17
- Maven Version: 3.9.3
- OS: Windows 10

**Comment by Hakky54** [comment]
Hi Emre, awesome and well discovered!
Would you like to provide the fix yourself as a pull request?

Please let me know whether you want to contribute or else I can also do the changes on my side : 😊

**Comment by eyny** [comment]
@Hakky54 I would like to, but I don't have the bandwidth. After the fix, it might be a good idea to create another release as the most recent version (v2.9.0) is 12 commits behind the master branch at the moment.

**Comment by Hakky54** [comment]
@eyny The issue has been resolved. It is now available in version 2.9.1
Thank you for reporting it!
